### PR TITLE
CN-6033 implemented open email dialog

### PR DESCRIPTION
**Pull request by srkela**
## Checklist (Mandatory):

- [x] I have named the PR correctly (Check PR1)
- [x] Check the commit’s or even all commits message styles matches our requested structure. (issue number, etc.) (Check CR10)
- [x] I have tested new code locally (Check CR3)
- [x] I have done a smoke test of the larger feature locally (Check CR3)
- [x] I have added at least two reviewers (Check PR5)
- [x] I have assigned the PR to myself (Check PR4)
- [x] I have merged develop (parent) branch into my feature branch and resolved all conflicts (Check PR7)

## Checklist (Optional):

- [ ] I have added necessary documentation, including instructions how to test (Check CR8)
- [ ] I have added tests that prove my fix/implementation is effective or that my feature works
- [ ] I have written the code according to the adopted style guide? (Check CR7)

## Additional notes

> What does this implement/fix? Explain your changes. (optional)
> Are there any scripts that need running (eg. yarn install)?
> Are there any edge cases that need checking?
> Are there any specific test instructions?

[MVPWorkshop Pull Request & Code Review Guidelines](https://docs.google.com/document/d/1TcuU1JBhlGZLtpy0osSNinXQ2piP_yrDuRcXdqaOS_U/edit#heading=h.5b6kpdr4btq8)

### CODE REVIEW

File: app/redux/userSecurity/userSecurityActions.js [hunk 1]
```diff
@@ -75,6 +77,7 @@ function disableTwoFactor() {
       await userSecurityService.disableTwoFactor(code);
       dispatch({ type: ACTIONS.DISABLE_TWO_FACTOR_SUCCESS });
       dispatch(navigateTo("SecuritySettings"));
+      openInbox();
```
Comment (srkela) [inline app/redux/userSecurity/userSecurityActions.js:80]: Remove

File: app/redux/wallet/walletActions.js [hunk 2]
```diff
@@ -135,12 +137,10 @@ function setCoinWithdrawalAddress(flow = "withdrawal") {
       dispatch(navigateTo(nextScreen));
 
       if (flow === "change-address") {
-        dispatch(
-          showMessage(
-            "success",
-            `Open your email to confirm the change of your ${formData.coin} withdrawal address. Note that withdrawals for ${formData.coin} will be locked for the next 24h due to our security protocols.`
-          )
-        );
+        await openInbox({
```
Comment (srkela) [inline app/redux/wallet/walletActions.js:140]: Remove await

File: app/components/screens/TransactionDetailsWithdraw/TransactionDetailsWithdraw.js [hunk 3]
```diff
@@ -60,25 +63,11 @@ class TransactionDetailsWithdraw extends Component {
...
-                  />
-                </View>
-                <CelText color={"white"} margin={"0 10 0 10"}>
-                  In order to proceed, you must confirm the transaction via
-                  email.
-                </CelText>
-              </View>
-            </InfoBox>
+            <CheckEmailInfoBox
+              infoText={
```
Comment (fjovakaric) [inline app/components/screens/TransactionDetailsWithdraw/TransactionDetailsWithdraw.js:67]: change copy
